soc/riscv/riscv-privilege/hi3861/CMakeLists.txt

```diff
@@ -0,0 +1,29 @@
+# Copyright (c) 2023 Chen Xingyu <hi@xingrz.me>
```

Comment:
What's the purpose of this script? Is it to program the device? If so - this should be moved to the board CMake script.

Reply:
The romcode of this SoC verifies the firmware on the flash, which must include a specific format of header (including some checksums). It is not a bare binary generated by objcopy -O binary. Also, it's not board-specific.